packages/generator/src/commonTemplateHelper.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Given an individual type or an array of types in the format Array\<Foo | Baa\>
+ * will return either the type prefixed by the namespace, or the Array with each type prefixed
+ * eg. Array\<types.Foo | types.Baa\>
+ *
+ * @param content - to be parsed for types to prefix with a namespace
+ * @param namespace - to be prefixed to types
+ * @returns the content prefixed with the namespace
+ */
+export function addNamespace(content: string, namespace: string): string {
+  // Not handling invalid content.
+  if (!content) {
+    throw new Error("Invalid content");
+  }
+  // Not handling invalid namespace.
+  if (!namespace) {
+    throw new Error("Invalid namespace");
+  }
+
+  // if the content is an array, extract all of the elements
+  let types = content;
+  let arrayType = false;
+  if (content.match("^Array") !== null) {
+    types = content.replace(/Array|<|>/g, "");
+    arrayType = true;
+  }
```

Comment:
```suggestion
  const matched = content.match(/^Array<(.*?)>$/);
  const arrayType = !!matched;
  const types = matched?.[1] || content;
```
1) `content.match()` should take a RegExp, not a string.
2) Since you're using RegExp, take advantage of match groups!